pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
-                            print("Unable to find tarkov-market page")
-                            print("="*80)
+        else:
+            logging.debug("In raid, no contour corrector")
+
+        if self.debug_mode >= 2:
+            cv2.imshow("final_image", image)
+            cv2.waitKey(0)
+
+        return image
+    
+    def validate_wordlist(self, wordlist):
```

Ruff (0.8.0)

Missing return type annotation for public function `validate_wordlist`

Add return type annotation: `bool`

(ANN201)